.kube/app/Dockerfile

```diff
@@ -48,6 +69,11 @@ RUN pecl install -o -f redis && \
     rm -rf /tmp/pear && \
     docker-php-ext-enable redis
 
+# - this can be removed once pecl fixes issues installing with php8.4
+RUN apt-get remove -y \
+    $PHPIZE_DEPS \
+    libtool
```

Comment:
Is it possible to do the remove in the same block as the install? I think it's more likely this will get missed when reverting later.

Reply:
It needs to be there or the following pecl build commands fail. I thought those commands ensures build dependencies were installed if necessary when pecl runs, but it failed when I removed them prior to running those.